### PR TITLE
Adding new workflow to test transformers-neuronx LLM on inf2

### PR DESCRIPTION
**Pull request by maaquib**
## Description ##

Adding a new workflow to test [transformers-neuronx](https://github.com/aws-neuron/transformers-neuronx) LLM on inf2

- If this change is a backward incompatible change, why must this change be made? N/A
- Interesting edge cases to note here N/A

### CODE REVIEW

File: .github/workflows/llm_inf2_integration.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+on:
+  workflow_dispatch:
+    inputs:
+      djl-version:
+        description: 'The released version of DJL'
+        required: false
+        default: ''
+  schedule:
+    - cron: '0 15 * * *'
```
[the remark this answers is not in the document]
Comment (maaquib) [reply .github/workflows/llm_inf2_integration.yml]: Done

**Review by lanking520 — APPROVED** [review]
